src/tools/annotation/FreehandMouseTool.js

```diff
@@ -980,6 +992,8 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
     }
 
     external.cornerstone.updateImage(element);
+
+    this.fireModifiedEvent(element, data);
```

Comment:
Thanks for adding these event fires, looks good to me.

Reply:
Thanks.